templates/HAPROXY_HEAD

```diff
@@ -0,0 +1,18 @@
+global
+  daemon
+  log /dev/log local0
+  log /dev/log local1 notice
+  maxconn 4096
+  tune.ssl.default-dh-param 2048
+defaults
+  log               global
+  retries           3
+  maxconn           2000
+  timeout connect   5s
+  timeout client    50s
+  timeout server    50s
+listen stats
+  bind 0.0.0.0:9090
```

Comment:
If you specify URIs in the Marathon JSON, Mesos will download the artifacts and place them into the Mesos sandbox. Mesos will then export the $MESOS_SANDBOX env var inside the Docker container, and mount the sandbox as a volume at that path.

See: https://mesos.apache.org/documentation/latest/docker-containerizer/ under the section "What does the Docker Containerizer do?"